### ISSUE TITLE
How do I get these WAC PS cmdlets?

### ISSUE THREAD

**Issue by namedtuple** [MicrosoftDocs/windowsserverdocs#3258]
I've tried installing both Windows Admin Center version 1904.1 and preview version 1908.  I don't see any of these PowerShell cmdlets available.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aafe0610-f679-83cc-b670-82a942be0fe2
* Version Independent ID: e4a9eb74-c3c6-e026-e5e7-189586c99528
* Content: [Install and Manage Extensions](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/configure/using-extensions#feedback)
* Content Source: [WindowsServerDocs/manage/windows-admin-center/configure/using-extensions.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/manage/windows-admin-center/configure/using-extensions.md)
* Product: **windows-server-threshold**
* Technology: **manage**
* GitHub Login: @daniellee-msft
* Microsoft Alias: **jol**

**Comment by daniellee-msft** [comment]
Did you try this: Import-Module "$env:ProgramFiles\windows admin center\PowerShell\Modules\ExtensionTools"

**Comment by namedtuple** [comment]
Ah..sorry.  The first line, thank you!

Do you know if there are any plans to have a way to manage UI Extensions via PowerShell?  Is there any way currently to install/uninstall a UI Extension without the Windows Admin Center GUI?

Thanks for your help.

**Comment by daniellee-msft** [comment]
In this doc, there are a few examples for install-extension. Are these what you are looking for?

**Comment by namedtuple** [comment]
I'm trying with the Install-Extension cmdlet but so far have not had luck.  We are developing a WAC extension and have built a nupkg by authoring a nuspec file and running nuget pack <nuspec_file>.  This successfully creates a nupkg file which I can then manually install via the WAC GUI.

Since we are still in early stages, we do not want to publish this package in a public place yet.

Is there any way to install the WAC extension via the command line when we have a local nupkg file?

**Comment by daniellee-msft** [comment]
Have you tried installing the package through the UI? I would suggest trying this first, to rule out whether it's a problem with the package itself or the PS cmdlet.

To install through the UI, you don't need to publish the package publicly. You can place it on a file share, or even just point to the local folder where the pkg is. In WAC Extension Manager, go to Feeds tab, add the file path as the feed URL, for example "c:\users\brian\extensions", and your package should show up in the "Available Extensions" tab.

**Comment by namedtuple** [comment]
Yes, installing the package through the UI does work.  But I would like to be able to install/uninstall the extension through a script.  Is this possible?  Thanks for the help.

**Comment by daniellee-msft** [comment]
Just tried right now and seems to work for me. Is this what you tried:
Install-Extension -GatewayEndpoint [gateway URL] [NuGet package ID] -Feed [feed URL]

For example, if you installed WAC on Win10 with default port:
Install-Extension -GatewayEndpoint "https://localhost:6516" "microsoft.security" -feed "c:\users\brian"

**Comment by namedtuple** [comment]
It works!  Thank you very much for your help.  I appreciate it!  Happy Friday.

**Comment by daniellee-msft** [comment]
You're welcome, glad to help :)

**Comment by IngridAtMicrosoft** [comment]
Thanks for your dedication to our documentation. If we have been unable to review your issue in a timely manner, we sincerely apologize for the delayed response. We have captured the informtion you have provided for review. We are closing this issue for now, but if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. #please-close
